src/Import/ZipImporter.php

```
@@ -3,9 +3,11 @@
 namespace Neusta\Pimcore\ImportExportBundle\Import;
 
 use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
 
 class ZipImporter
 {
+    private const string ZIPIMPORT_DIR_PREFIX = 'zipimport_';
```

PHPStan (2.1.15)

Class constants with native types are supported only on PHP 8.3 and later.


(classConstant.nativeTypeNotSupported)